qml/Main.qml

```diff
@@ -119,10 +119,8 @@ Item {
             Keyboard {
                 id: vkb
 
-                property bool visibleSetting: true
-
                 y: parent.height-vkb.height
-                visible: page.activeFocus && visibleSetting
+                visible: page.activeFocus && util.keyboardMode != Util.KeyboardOff
```

Comment:
```suggestion
                visible: page.activeFocus && util.keyboardMode !== Util.KeyboardOff
```
It's not essential, but it'd be good to use `!==` and `===` for the conditionals.

Reply:
Sure.